src/webhooks/webhook.ts

```diff
@@ -32,17 +32,24 @@ async function triggerAutomation(payload, record) {
   }
 
   const triggers: Record<string, (pr: any) => string> = {
-    closed: (pr) => (pr.merged ? "prMerged" : "prClosed"),
-    opened: (pr) => (pr.draft ? "draftPrOpened" : "prOpened"),
+    closed: (payload) => (payload.pull_request.merged ? "prMerged" : "prClosed"),
+    opened: (payload) => (payload.pull_request.draft ? "draftPrOpened" : "prOpened"),
     reopened: () => "prReopened",
+    submitted: (payload) => {
+      switch (payload.review.state) {
+        case 'approved':
+          return "prApproved"
+        case 'changes_requested':
+          return "prChangesRequested"
+        default:
+          return ""
+      }
+    }
   };
 
-  const trigger = (triggers[payload.action] || (() => null))(
-    payload.pull_request
-  );
+  const trigger = (triggers[payload.action] || (() => null))(payload);
 
   if (trigger) {
-    console.log("Automation trigger");
```

Comment:
Can we leave this in? We keep removing the console logs and then when something breaks its really hard to diagnose the problem - these go to the invocation logs. General philosophy at Aha! is that it's better to log too much than too little.

Reply:
I don't mind to, yea. I only pulled it because the system is already adding those logs.

<img width="590" alt="Screen Shot 2022-07-28 at 8 12 28 AM" src="https://user-images.githubusercontent.com/104908/181502016-ae7dabda-a9c4-4983-8c5e-3d3916a124f8.png">

Reply:
Restored!

Reply:
thanks. yeah its logging it when you make the call to Aha! but I think its a good idea to log it also from the extension that if it doesn't fire we can see where the issue lies